lib/resty/dns/resolver.lua

```diff
@@ -99,6 +99,26 @@ for i = 2, 64, 2 do
     arpa_tmpl[i] = DOT_CHAR
 end
 
+local function udp_socks_close(self)
```

Comment:
this file uses two blank lines to seperate functions.

Reply:
done